src/ViewModel/ContentHeader.php

```diff
@@ -78,6 +88,19 @@ public function __construct(
         $this->audioPlayer = $audioPlayer;
     }
 
+    private static function designateTitleLength($title) : string
```

Comment:
@thewilkybarkid `CarouselItem` needs to use this, too. Do you think `designateTitleLength()` should be extracted, along with its associated `const`s `LENGTH_LIMITS` and `MAX_LENGTH` into a `trait`?